mautic/mautic
# Bugfix for FingerprintJS

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Including `mtc.js` on pages that have RequireJS results in FingerprintJS not being defined.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Embed mtc.js on a website that uses RequireJS
2. Notice the error in your browser's developer console.

#### Steps to test this PR:
1. Apply PR
2. Redo reproduction tests
3. Error will be gone
4. Ensure that new leads are still created, and that the incoming request to Mautic includes an appropriate fingerprint value (to ensure that FingerprintJS is still working)
